### DISCUSSION TITLE
Make the Telemetry window fit the 20 drivers from top to bottom instead of 10 on the left and 10 on the right

### DISCUSSION THREAD

**Discussion by RCD800** [f1multiviewer/issue-tracker#67] [Feature requests] [answered]
So, would be cool to when resized the window, it adjusted the left and right drivers to be all "on the left"
![muurtu](https://user-images.githubusercontent.com/85562954/174597242-a13e83a2-ff48-4e76-b92e-32f6626459a9.png)

**Comment by JJWatMyself** [comment]
I agree with  this and if user resizes window double wide, would flow those 11-20 to the right side as they are in the original layout.

In addition, I am not a big fan of the stacking of stats and think they should be side-by-side. The problem is as you scan your eye and up and down the list for GAP, you are looking for gaps <1 second and it is harder to spot those when the LEADER number is like a leap frog for the eye to deal with. So to take this window width concept one step further, as the window width increases, I would like to see those double stacked stats for BEST, LAP, GAP LEADER to be on a single row and not double stacked.

Quick version of what this looks like below. 

The order might be better to be LAST GAP LEADER BEST

![Untitled](https://user-images.githubusercontent.com/22590445/175193795-45848ba5-0f29-4276-b39b-34c63a2c9269.png)

**Comment by f1multiviewer** [comment, marked as answer]
<img width="830" alt="image" src="https://user-images.githubusercontent.com/106821994/176993263-f3510c16-cdcf-425d-814e-ac1b461c3aff.png">

Done, will now automatically change when < 1400px width
